scripts/add_framework_script.rb

```diff
@@ -65,6 +65,7 @@ def add_ref(group, path, source_tree, phase_list)
       framework_set = project_target.frameworks_build_phase.files.to_set
       puts "The following frameworks are added to #{project_target}"
       embed_frameworks_phase = project_target.new_copy_files_build_phase("Embed Frameworks")
+      embed_frameworks_phase.dst_subfolder_spec = "10" # `Frameworks` directory
```

Comment:
Why 10?

Reply:
Discussed offline. It appears like there is an enum of sorts where numbers map to the directory (10 for Frameworks, 7 for Resources).